Mailer/TwigSwiftMailer.php

```diff
@@ -46,21 +46,26 @@ class TwigSwiftMailer implements MailerInterface
     private $from;
 
     /**
+     * @var array
+     */
+    private $to;
+
+    /**
      * Constructor
      *
      * @param \Swift_Mailer     $mailer           A mailer instance
      * @param \Twig_Environment $twig             A Twig instance
-     * @param string            $recipientAddress The recipient email
      * @param string            $template         The template used for email content
      * @param array             $from             The From address
+     * @param array             $to               The To address
      */
-    public function __construct(\Swift_Mailer $mailer, \Twig_Environment $twig, $recipientAddress, $template, array $from = array())
+    public function __construct(\Swift_Mailer $mailer, \Twig_Environment $twig, $template, array $from = array(), array $to = array())
```

Comment:
Argument `to` is not optional but mandatory. Can you remove the default value and move it before the `from` please?